### PR TITLE
list media

### PR DESCRIPTION
**Pull request by huyencao**
https://edu-redmine.sun-asterisk.vn/issues/10953

### CODE REVIEW

**Comment by huyencao** [comment]
ready

File: resources/views/backend/layouts/header.blade.php [hunk 1]
```diff
@@ -10,29 +10,18 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a href="#" class="navbar-brand"> {{ __('Bootflat-Admin') }} </a>
+            <a href="{{ route('dashboard')}}" class="navbar-brand"> {{ __('Tourist') }} </a>
```
Comment (BunnyPi04) [inline resources/views/backend/layouts/header.blade.php]: convention

File: resources/views/backend/layouts/sidebar.blade.php [hunk 2]
```diff
@@ -2,22 +2,18 @@
     <ul class="list-group panel">
         <li class="list-group-item"><i class="glyphicon glyphicon-align-justify"></i> <b> {{ __('DANH MỤC QUẢN
                 LÝ') }} </b></li>
-        <li class="list-group-item"><a href="index.html"><i class="glyphicon glyphicon-home"></i> {{ __('Trang chủ') }}
+        <li class="list-group-item"><a href="{{ route('dashboard' )}}"><i class="glyphicon glyphicon-home"></i> {{ __('Trang chủ') }}
```
Comment (BunnyPi04) [inline resources/views/backend/layouts/sidebar.blade.php]: convention

File: resources/views/backend/media/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <div class="row">
+                <div class="col-md-12">
+                    <h3 class="panel-title"><a href="javascript:void(0);" class="toggle-sidebar"><span
+                                class="fa fa-angle-double-left" data-toggle="offcanvas"
+                                title="{{ __('Maximize Panel') }}"></span></a> {{ __('List media') }} </h3>
+                </div>
+            </div>
+        </div>
+        <div class="panel-body">
+                <div class="content-row">
```
Comment (BunnyPi04) [inline resources/views/backend/media/index.blade.php]: indent = 4 spaces nhé

**Pull request merged**